packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_controller.dart

```diff
@@ -44,6 +48,12 @@ class PropertyEditorController extends DisposableController
           position: cursorPosition,
         );
         final args = result?.args ?? <EditableArgument>[];
+        // Register impression.
+        ga.impression(
+          gaId,
+          gac.PropertyEditorSidebar.widgetPropertiesUpdate.name,
+          // TODO(): Add metrics for widget name.
```

Comment:
We have to be careful with the number of metrics we add. Maybe we need a generic field where we can pass an extra bit of data that we can re-use for several analytics events.

Reply:
Updated the TODO to specify postfixing the widget name to this event so that we aren't adding more metrics